src/api.ts

```diff
@@ -340,6 +397,8 @@ type CreateAttachmentParams = {
  * Then you can use the `api` object to make calls to the Literal service.
  */
 export class API {
+  /** @ignore */
+  public cache: SharedCache;
```

Comment:
This should be internal

Reply:
used it to simplify my tests implementation and forgot to change it later